promscale/quick-start.md

```diff
@@ -6,40 +6,31 @@ Jaeger and OpenTelemetry. It also includes some additional pre-configured toolin
 to get familiar with additional Promscale features, such as application performance
 monitoring dashboards and visualization tools.
 
-To learn more about how Promscale can improve your observability stack, see
-[About Promscale][about-promscale]. For instructions on installing Promscale in your
```

Comment:
I think it's important to leave this in, because readers are relatively likely to hit this quick start first when they come to Promscale docs. If you don't know what you're doing, this quick start is a bit confronting, so if they're staring at this page thinking "Sure, but the what hell even is Promscale, and do I want it?" they need a sign post.

Reply:
We are doing some experiments, trying to understand what works better (let our users and outcomes guide our decisions). It will be easy to continue iterating and go back if it doesn't work.

Let's give it a go and we'll see.